libs/extensions-registry/tinymce-plugin.model.ts

```diff
@@ -0,0 +1,13 @@
+import type { ClassConstructor } from "@umbraco-cms/backoffice/models";
+import { TinyMcePluginArguments, TinyMcePluginBase } from "../../src/backoffice/shared/property-editors/uis/tiny-mce/plugins/tiny-mce-plugin";
```

Comment:
Please make sure that all models needed in a lib resides in the libs folder somewhere, otherwise the type definitions are not going to work, nor is the library loadable on the v13/dev branch.